src/core/filesystem.cc

```diff
@@ -670,8 +672,8 @@ GCSFileSystem::ReadTextFile(const std::string& path, std::string* contents)
 
   if (!stream) {
     return Status(
-        Status::Code::INTERNAL,
-        "Failed to open object read stream for " + path);
+        Status::Code::INTERNAL, "Failed to open object read stream for " +
+                                    path + " : " + stream.status().message());
```

Comment:
"stream" is defined even if !stream ?

Reply:
Yes. I followed the examples [here](https://github.com/googleapis/google-cloud-cpp/blob/master/google/cloud/storage/examples/storage_bucket_samples.cc) and [here](https://googleapis.dev/cpp/google-cloud-storage/latest/).